### DISCUSSION TITLE
ESPResSo++ weekly meeting (4.00 pm, 26. Jan)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#477] [Team Posts]
Thema: ESPResSo++ Developer Meeting
Uhrzeit: 15.Dez. 2022 04:00 PM Amsterdam, Berlin, Rom, Stockholm, Wien

Zoom-Meeting beitreten
https://zoom.us/j/99468001371?pwd=RWVGUmRIQXlWdUFMRnNnM0loZnpyQT09

Meeting-ID: 994 6800 1371
Kenncode: 537959
Schnelleinwahl mobil
+16892781000,,99468001371#,,,,*537959# Vereinigte Staaten
+17193594580,,99468001371#,,,,*537959# Vereinigte Staaten

**Comment by mrampp** [comment]
apologies, there is a conflict with another meeting tomorrow, i.e.
I cannot join this time.
I'll be there next week

Markus
